Modelica/Mechanics/MultiBody/Parts.mo

```diff
@@ -2610,18 +2598,18 @@ November 3-4, 2003, pp. 149-158
 
     parameter SI.Distance width=0.035 "Width of wheel" annotation (Dialog(
         tab="Animation",
-        group="if animation = true",
+        group="If animation = true",
```

Comment:
No need to touch the Dialo.group annotation at all. It is "if animation = true" also in Sensors and Joints.

Reply:
OK, I will revert this. 
But generally I like using upper case in groups and tabs, so may be I willl change it all in the future.

Reply:
I think it is intended to look the conditional declaration itself - thus "if" is a keyword explaining the lower case.

Reply:
I see. 

I guess it's anyway questionable if such a Group's title is reasonable:
1. The parameters are all collected in a separate Tab called "Animation". So what else is expected there?
2. What if I want to further sort animation parameters. Let say by a considering body. How should the Group's title look like?
Example:
   - parameter MBS.Color colorBody1 "Color" annotation(Dialog(group="Body 1"));
   - parameter MBS.Color colorBody2 "Color" annotation(Dialog(group="Body 2"));

    Do I have to repeat "if animation = true" again and again? I.e. group="Body 1, if animation = true" or something like this?

Another option is to move the parameter "animation" into the same Tab. But then it would become cumbersome to switch the animation on and off.